wiki/markdown/docker.md

```diff
@@ -87,10 +87,10 @@ This docker-compose file includes these components, [docker-compose.yml](https:/
 - MongoDb
 - RabbitMq
 - ER
-- EI-Backend
```

Comment:
I think the component names can use "backend" and "frotend" format and when describing something in text we can use "back-end" and "front-end" format.

Reply:
I'm thinking it might confuse readers, since this entire file is meant for documentation purposes. An alternative could be to refer to the actual names used in the docker-compose file, but are we sure these names won't change in the future? I'm sticking with this for now to follow the documentation style.

Reply:
Since line above refer to docker-compose file, my mind is reading as how it is named in docker-compose file.
If the service name changes completely in docker-compose file, then documentation need to be updated as well.